src/main/java/org/kainos/ea/team2/db/DatabaseConnector.java

```diff
@@ -23,11 +23,6 @@ public static Connection getConnection() throws SQLException, IllegalArgumentExc
             throw new IllegalArgumentException("Environment variables not set");
         }
 
-
-        if(user == null || password == null || host == null) {
-            throw new IllegalArgumentException("Properties file must exist and must contain user, password, name and host properties");
-        }
```

Comment:
any reason for removing this check?

Reply:
There was another one in there, merge conflict caused two of them to be in the same file